chore: migrate react-text to use Griffel

layershifter opened the pull request:
## PR changes

This PR replaces `@fluentui/react-make-styles` with `@griffel/core` and related packages in `@fluentui/react-text` package.

Following packages were replaced:

- `@fluentui/react-make-styles` => `@griffel/react`
- `@fluentui/babel-make-styles` => `@griffel/babel-preset`
- `@fluentui/jest-serializer-make-styles` => `@griffel/jest-serializer`
- `@fluentui/react-conformance-make-styles` => `@fluentui/react-conformance-griffel`

---

Note: check #21360 for more details about changes.

fabricteam commented:
## 📊 Bundle size report

| Package & Exports                                                                                        |    Baseline (minified/GZIP) |                          PR |                                                                                                                                                                                                                                     Change |
| :------------------------------------------------------------------------------------------------------- | --------------------------: | --------------------------: | -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>   | `10.837 kB`<br />`4.248 kB` | `10.769 kB`<br />`4.224 kB` | `-68 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/Decrease.svg" /><br />`-24 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/Decrease.svg" /> |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr> | `14.139 kB`<br />`4.589 kB` | `14.077 kB`<br />`4.565 kB` | `-62 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/Decrease.svg" /><br />`-24 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/Decrease.svg" /> |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `48.475 kB`<br />`14.343 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |  `43.984 kB`<br />`13.191 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |    `7.686 kB`<br />`3.254 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |    `8.953 kB`<br />`3.693 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |    `7.879 kB`<br />`3.381 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `166.076 kB`<br />`47.058 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `32.489 kB`<br />`10.636 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/b1446dead62e61c9a0f6101d6dfc5a2f780403b8'>b1446dead62e61c9a0f6101d6dfc5a2f780403b8</a></sub>
<div id="bundleSizeComment9423"/>

fabricteam commented:
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/FluentProviderWithTheme-mount_base.html">123</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/FluentProviderWithTheme-mount.html">175</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/FluentProviderWithTheme-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Avatar-mount_base.html">770</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Avatar-mount.html">788</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/BaseButton-mount_base.html">799</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/BaseButton-mount.html">790</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Breadcrumb-mount_base.html">2248</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Breadcrumb-mount.html">2288</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/ButtonNext-mount_base.html">445</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/ButtonNext-mount.html">447</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Checkbox-mount_base.html">1352</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Checkbox-mount.html">1324</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/CheckboxBase-mount_base.html">1120</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/CheckboxBase-mount.html">1129</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/ChoiceGroup-mount_base.html">4123</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/ChoiceGroup-mount.html">4143</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/ComboBox-mount_base.html">833</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/ComboBox-mount.html">864</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/CommandBar-mount_base.html">8825</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/CommandBar-mount.html">8900</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/ContextualMenu-mount_base.html">7188</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/ContextualMenu-mount.html">7245</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/DefaultButton-mount_base.html">998</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/DefaultButton-mount.html">999</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/DetailsRow-mount_base.html">3260</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/DetailsRow-mount.html">3327</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/DetailsRowFast-mount_base.html">3271</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/DetailsRowFast-mount.html">3251</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/DetailsRowNoStyles-mount_base.html">3110</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/DetailsRowNoStyles-mount.html">3123</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Dialog-mount_base.html">2229</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Dialog-mount.html">2269</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/DocumentCardTitle-mount_base.html">159</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/DocumentCardTitle-mount.html">156</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Dropdown-mount_base.html">2797</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Dropdown-mount.html">2761</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/FluentProviderNext-mount_base.html">1712</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/FluentProviderNext-mount.html">1799</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/FluentProviderWithTheme-mount_base.html">123</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/FluentProviderWithTheme-mount.html">175</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/FluentProviderWithTheme-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/FluentProviderWithTheme-virtual-rerender_base.html">95</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/FluentProviderWithTheme-virtual-rerender.html">95</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">163</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.html">173</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/FocusTrapZone-mount_base.html">1593</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/FocusTrapZone-mount.html">1627</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/FocusZone-mount_base.html">1560</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/FocusZone-mount.html">1583</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/IconButton-mount_base.html">1508</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/IconButton-mount.html">1520</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Label-mount_base.html">310</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Label-mount.html">318</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Layer-mount_base.html">2586</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Layer-mount.html">2579</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Link-mount_base.html">422</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Link-mount.html">448</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/MakeStyles-mount_base.html">1477</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/MakeStyles-mount.html">1482</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/MenuButton-mount_base.html">1294</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/MenuButton-mount.html">1270</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/MessageBar-mount_base.html">1731</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/MessageBar-mount.html">1821</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Nav-mount_base.html">2858</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Nav-mount.html">2855</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/OverflowSet-mount_base.html">1001</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/OverflowSet-mount.html">961</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Panel-mount_base.html">2120</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Panel-mount.html">2165</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Persona-mount_base.html">757</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Persona-mount.html">751</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Pivot-mount_base.html">1256</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Pivot-mount.html">1277</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/PrimaryButton-mount_base.html">1111</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/PrimaryButton-mount.html">1117</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Rating-mount_base.html">6630</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Rating-mount.html">6577</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/SearchBox-mount_base.html">1160</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/SearchBox-mount.html">1146</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Shimmer-mount_base.html">2174</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Shimmer-mount.html">2185</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Slider-mount_base.html">1705</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Slider-mount.html">1710</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/SpinButton-mount_base.html">4258</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/SpinButton-mount.html">4248</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Spinner-mount_base.html">383</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Spinner-mount.html">393</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/SplitButton-mount_base.html">2696</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/SplitButton-mount.html">2735</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Stack-mount_base.html">476</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Stack-mount.html">464</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/StackWithIntrinsicChildren-mount_base.html">2005</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/StackWithIntrinsicChildren-mount.html">1996</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/StackWithTextChildren-mount_base.html">4533</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/StackWithTextChildren-mount.html">4513</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/SwatchColorPicker-mount_base.html">9832</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/SwatchColorPicker-mount.html">9812</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/TagPicker-mount_base.html">2291</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/TagPicker-mount.html">2287</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/TeachingBubble-mount_base.html">11267</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/TeachingBubble-mount.html">11345</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Text-mount_base.html">395</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Text-mount.html">387</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/TextField-mount_base.html">1216</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/TextField-mount.html">1216</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/ThemeProvider-mount_base.html">1061</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/ThemeProvider-mount.html">1030</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/ThemeProvider-virtual-rerender_base.html">552</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/ThemeProvider-virtual-rerender.html">543</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1606</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1585</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Toggle-mount_base.html">724</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/Toggle-mount.html">721</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/buttonNative-mount_base.html">136</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test/buttonNative-mount.html">143</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

## Perf Analysis (`@fluentui/react-northstar`)
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
IconMinimalPerf.default | [537](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/IconMinimalPerf.default.html) | [492](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/IconMinimalPerf.default_base.html) | 1.09:1
AvatarMinimalPerf.default | [173](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/AvatarMinimalPerf.default.html) | [167](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 1.04:1
BoxMinimalPerf.default | [303](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/BoxMinimalPerf.default.html) | [290](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/BoxMinimalPerf.default_base.html) | 1.04:1
RefMinimalPerf.default | [213](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/RefMinimalPerf.default.html) | [205](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/RefMinimalPerf.default_base.html) | 1.04:1
TableMinimalPerf.default | [351](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/TableMinimalPerf.default.html) | [339](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/TableMinimalPerf.default_base.html) | 1.04:1
TreeWith60ListItems.default | [155](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/TreeWith60ListItems.default.html) | [149](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/TreeWith60ListItems.default_base.html) | 1.04:1
AnimationMinimalPerf.default | [473](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/AnimationMinimalPerf.default.html) | [458](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 1.03:1
MenuMinimalPerf.default | [732](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/MenuMinimalPerf.default.html) | [708](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/MenuMinimalPerf.default_base.html) | 1.03:1
StatusMinimalPerf.default | [583](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/StatusMinimalPerf.default.html) | [566](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/StatusMinimalPerf.default_base.html) | 1.03:1
AccordionMinimalPerf.default | [135](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/AccordionMinimalPerf.default.html) | [132](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 1.02:1
ChatDuplicateMessagesPerf.default | [267](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [263](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 1.02:1
DialogMinimalPerf.default | [653](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/DialogMinimalPerf.default.html) | [643](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/DialogMinimalPerf.default_base.html) | 1.02:1
FormMinimalPerf.default | [343](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/FormMinimalPerf.default.html) | [335](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/FormMinimalPerf.default_base.html) | 1.02:1
HeaderSlotsPerf.default | [656](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/HeaderSlotsPerf.default.html) | [646](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 1.02:1
ItemLayoutMinimalPerf.default | [1006](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [988](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 1.02:1
LabelMinimalPerf.default | [335](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/LabelMinimalPerf.default.html) | [330](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/LabelMinimalPerf.default_base.html) | 1.02:1
ListMinimalPerf.default | [444](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ListMinimalPerf.default.html) | [437](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ListMinimalPerf.default_base.html) | 1.02:1
RosterPerf.default | [1023](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/RosterPerf.default.html) | [1006](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/RosterPerf.default_base.html) | 1.02:1
ReactionMinimalPerf.default | [328](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ReactionMinimalPerf.default.html) | [322](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 1.02:1
SliderMinimalPerf.default | [1436](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/SliderMinimalPerf.default.html) | [1405](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/SliderMinimalPerf.default_base.html) | 1.02:1
TextMinimalPerf.default | [307](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/TextMinimalPerf.default.html) | [300](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/TextMinimalPerf.default_base.html) | 1.02:1
ButtonOverridesMissPerf.default | [1441](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [1432](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 1.01:1
ChatWithPopoverPerf.default | [312](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ChatWithPopoverPerf.default.html) | [308](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 1.01:1
DividerMinimalPerf.default | [312](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/DividerMinimalPerf.default.html) | [309](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/DividerMinimalPerf.default_base.html) | 1.01:1
DropdownManyItemsPerf.default | [581](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/DropdownManyItemsPerf.default.html) | [577](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 1.01:1
GridMinimalPerf.default | [295](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/GridMinimalPerf.default.html) | [291](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/GridMinimalPerf.default_base.html) | 1.01:1
HeaderMinimalPerf.default | [311](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/HeaderMinimalPerf.default.html) | [307](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 1.01:1
InputMinimalPerf.default | [1107](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/InputMinimalPerf.default.html) | [1097](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/InputMinimalPerf.default_base.html) | 1.01:1
LayoutMinimalPerf.default | [317](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/LayoutMinimalPerf.default.html) | [314](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 1.01:1
LoaderMinimalPerf.default | [588](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/LoaderMinimalPerf.default.html) | [581](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 1.01:1
PopupMinimalPerf.default | [527](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/PopupMinimalPerf.default.html) | [524](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/PopupMinimalPerf.default_base.html) | 1.01:1
TableManyItemsPerf.default | [1620](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/TableManyItemsPerf.default.html) | [1609](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/TableManyItemsPerf.default_base.html) | 1.01:1
VideoMinimalPerf.default | [548](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/VideoMinimalPerf.default.html) | [543](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/VideoMinimalPerf.default_base.html) | 1.01:1
ButtonSlotsPerf.default | [464](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ButtonSlotsPerf.default.html) | [464](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 1:1
ChatMinimalPerf.default | [627](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ChatMinimalPerf.default.html) | [629](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ChatMinimalPerf.default_base.html) | 1:1
CheckboxMinimalPerf.default | [2279](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/CheckboxMinimalPerf.default.html) | [2271](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 1:1
DropdownMinimalPerf.default | [2560](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/DropdownMinimalPerf.default.html) | [2561](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 1:1
FlexMinimalPerf.default | [249](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/FlexMinimalPerf.default.html) | [249](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/FlexMinimalPerf.default_base.html) | 1:1
ListNestedPerf.default | [470](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ListNestedPerf.default.html) | [472](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ListNestedPerf.default_base.html) | 1:1
ProviderMergeThemesPerf.default | [1474](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [1473](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 1:1
ProviderMinimalPerf.default | [970](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ProviderMinimalPerf.default.html) | [973](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 1:1
SegmentMinimalPerf.default | [300](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/SegmentMinimalPerf.default.html) | [299](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 1:1
CustomToolbarPrototype.default | [3496](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/CustomToolbarPrototype.default.html) | [3501](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 1:1
ButtonMinimalPerf.default | [148](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ButtonMinimalPerf.default.html) | [149](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 0.99:1
CardMinimalPerf.default | [465](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/CardMinimalPerf.default.html) | [470](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/CardMinimalPerf.default_base.html) | 0.99:1
EmbedMinimalPerf.default | [3479](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/EmbedMinimalPerf.default.html) | [3503](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 0.99:1
ListCommonPerf.default | [535](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ListCommonPerf.default.html) | [538](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ListCommonPerf.default_base.html) | 0.99:1
MenuButtonMinimalPerf.default | [1413](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [1425](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 0.99:1
PortalMinimalPerf.default | [154](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/PortalMinimalPerf.default.html) | [155](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/PortalMinimalPerf.default_base.html) | 0.99:1
SkeletonMinimalPerf.default | [297](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/SkeletonMinimalPerf.default.html) | [300](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 0.99:1
SplitButtonMinimalPerf.default | [3640](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [3665](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 0.99:1
TextAreaMinimalPerf.default | [420](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/TextAreaMinimalPerf.default.html) | [426](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 0.99:1
TooltipMinimalPerf.default | [877](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/TooltipMinimalPerf.default.html) | [886](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 0.99:1
TreeMinimalPerf.default | [674](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/TreeMinimalPerf.default.html) | [681](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/TreeMinimalPerf.default_base.html) | 0.99:1
CarouselMinimalPerf.default | [392](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/CarouselMinimalPerf.default.html) | [400](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 0.98:1
ImageMinimalPerf.default | [317](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ImageMinimalPerf.default.html) | [322](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ImageMinimalPerf.default_base.html) | 0.98:1
ListWith60ListItems.default | [545](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ListWith60ListItems.default.html) | [555](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ListWith60ListItems.default_base.html) | 0.98:1
RadioGroupMinimalPerf.default | [377](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [386](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 0.98:1
AlertMinimalPerf.default | [229](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/AlertMinimalPerf.default.html) | [235](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/AlertMinimalPerf.default_base.html) | 0.97:1
AttachmentSlotsPerf.default | [900](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/AttachmentSlotsPerf.default.html) | [925](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 0.97:1
DatepickerMinimalPerf.default | [4589](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/DatepickerMinimalPerf.default.html) | [4733](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 0.97:1
ToolbarMinimalPerf.default | [787](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ToolbarMinimalPerf.default.html) | [809](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 0.97:1
AttachmentMinimalPerf.default | [132](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/AttachmentMinimalPerf.default.html) | [137](https://fluentuipr.z22.web.core.windows.net/pull/21418/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 0.96:1

</details><div id="perfComment9423"/>

andrefcdias commented:
nit: if possible, update the MIGRATION.md too in this PR

layershifter commented:
> nit: if possible, update the MIGRATION.md too in this PR

- updated snippets to fix syntax errors & remove `theme`
- add imports to make snippets copy-pastable
- replaced `make-styles` mentions with Griffel

@andrefcdias can you please check changes to `MIGRATION.md`?
